### PR TITLE
Fixes #9299 - Add wimboot template to WAIK

### PR DESCRIPTION
**Pull request by kireevco**
Adding a windows template that works with wimboot. If anyone was able to make the old template work without WDS - let me know.

### CODE REVIEW

File: waik/PXELinux.erb [hunk 1]
```diff
@@ -2,6 +2,9 @@
...
 %>
-default winPE
-label winPE
-kernel /Boot/startrom.0
+default wimboot
+label wimboot
+    menu default
+     menu label Wimboot via pxelinux
+     com32 linux.c32 <%= @kernel %>
+     append initrdfile=<%= @initrd %>,<%= @host.operatingsystem.bootfile(@host.arch,:bcd) %>,<%= @host.operatingsystem.bootfile(@host.arch,:bootsdi) %>,<%= @host.operatingsystem.bootfile(@host.arch,:bootwim) %>
```
Comment (ekohl) [inline waik/PXELinux.erb]: Nitpick: generally we prefer line endings at the end of the file.

**Comment by ekohl** [comment]
I don't have knowledge of windows and pxelinux, so unless someone else can provide some insight I'm tempted to trust you and merge it.

**Comment by domcleal** [comment]
It can't be worse than what we have :smile:

**Comment by kireevco** [comment]
Great, thank you. 
In order everything to work we need to push on 
https://github.com/theforeman/foreman/pull/2143 and https://github.com/theforeman/smart-proxy/pull/257

File: waik/PXELinux.erb [hunk 2]
```diff
@@ -2,6 +2,9 @@
 kind: PXELinux
 name: WAIK default PXELinux
 %>
-default winPE
-label winPE
-kernel /Boot/startrom.0
+default wimboot
+label wimboot
+    menu default
+     menu label Wimboot via pxelinux
```
Comment (ekohl) [inline waik/PXELinux.erb]: I recall PXELinux had some issue with two menu items, but not sure of the details. Does this work?

**Comment by kireevco** [comment]
Can we please merge this? Is there anything else required? https://github.com/theforeman/smart-proxy/pull/257 & https://github.com/theforeman/foreman/pull/2143 are ready.

**Comment by ekohl** [comment]
I have 2 inline notes, if you can address those I think it can be merged.

File: waik/PXELinux.erb (hunk 1 above)
Comment (DimmKirr) [inline waik/PXELinux.erb]: Fixed.

File: waik/PXELinux.erb (hunk 2 above)
Comment (DimmKirr) [inline waik/PXELinux.erb]: I added some changes. The idea that it should render to something like that:

```
default wimbootpe
label wimbootpe
    menu default
     menu label Wimboot via pxelinux
     com32 linux.c32 boot/windows-x64/wimboot
     append initrdfile=boot/windows-x64/bootmgr,boot/windows-x64/bcd,boot/windows-x64/boot.sdi,boot/windows-x64/boot.wim
```

**Comment by mmoll** [comment]
@kireevco ping? As said in IRC, I think/hope this could work completely wihout menu at all.

**Comment by DimmKirr** [comment]
I'm out these 2 days. I'll test it without menus next week .

**Comment by mmoll** [comment]
take your time :)

**Comment by ares** [comment]
@kireevco could you please reopen against develop branch? we renamed master to develop since we started branching this repo per foreman releases

**Pull request closed without merging**